projects/kit/components/select/select-option/select-option.component.ts

```diff
@@ -0,0 +1,63 @@
+import {ChangeDetectionStrategy, Component, computed, inject} from '@angular/core';
+import {TuiControl} from '@taiga-ui/cdk/classes';
+import {tuiDirectiveBinding, tuiIsPresent} from '@taiga-ui/cdk/utils/miscellaneous';
+import type {TuiDataListHost} from '@taiga-ui/core/components/data-list';
+import {
+    TUI_DATA_LIST_HOST,
+    TuiOptionWithValue,
+} from '@taiga-ui/core/components/data-list';
+import {TuiScrollIntoView} from '@taiga-ui/core/components/scrollbar';
+import {tuiInjectAuxiliary} from '@taiga-ui/core/components/textfield';
+import {TuiIcons} from '@taiga-ui/core/directives/icons';
+import {TuiItemsHandlersDirective} from '@taiga-ui/core/directives/items-handlers';
+import {TUI_COMMON_ICONS} from '@taiga-ui/core/tokens';
+
+@Component({
+    standalone: true,
+    selector: 'tui-select-option',
```

Comment:
Do we need it? I think it will never be used.

Reply:
https://github.com/taiga-family/taiga-ui/pull/10642/commits/f83d1f83aa343cdf442eebaf72b4d169d1fdc43a